src/com/epam/javatrainings/annotations/forAdulthood/AdulthoodProcessor.java

```diff
@@ -0,0 +1,52 @@
+package com.epam.javatrainings.annotations.forAdulthood;
+
+import com.epam.javatrainings.annotations.exceptions.NotAdulthoodException;
+import com.epam.javatrainings.annotations.util.ClassFinder;
+
+import java.io.IOException;
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+
+public class AdulthoodProcessor {
+    public void doProcess(String scanPackage, LocalDate localDate) throws IllegalAccessException, IOException,
```

Comment:
you need to pass a dto not can a package.

Reply:
Ok, understood. But if I have a few variables annotated in the same package but in different classes, it will work.